.github/workflows/issue.yml

```diff
@@ -24,12 +24,30 @@ jobs:
         summary: « [hugr] ${{ github.event.issue.title }}»
         description: ${{ github.event.issue.html_url }}
         fields: '{"labels": ["hugr"]}'
+    - name: set_priority_var
+      env:
+        GH_PRIORITY: ${{ github.event.issue.label }}
```

Comment:
I think there's only a "labels" array?
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=opened#issues